src/tcptunnel.c

```diff
@@ -41,14 +41,15 @@
 
 struct struct_rc rc;
 struct struct_options options;
-struct struct_settings settings = { 0, 0, 0, 0, 0, 0, 0, 0 };
+struct struct_settings settings = { 0, 0, 0, 0, 0, 0, 0, 0, 0 };
 
 static struct option long_options[] = {
 	{ "local-port",    required_argument, NULL, LOCAL_PORT_OPTION },
 	{ "remote-host",   required_argument, NULL, REMOTE_HOST_OPTION },
 	{ "remote-port",   required_argument, NULL, REMOTE_PORT_OPTION },
 	{ "bind-address",  required_argument, NULL, BIND_ADDRESS_OPTION },
 	{ "buffer-size",   required_argument, NULL, BUFFER_SIZE_OPTION },
+	{ "client-addr",   required_argument, NULL, CLIENT_ADDR_OPTION },
```

Comment:
Can you rename `CLIENT_ADDR_OPTION` into `CLIENT_ADDRESS_OPTION`? So it matches the naming of `BIND_ADDRESS_OPTION`.